lib/ex_auction/router.ex

```diff
@@ -11,6 +11,9 @@ defmodule ExAuction.Router do
 
   plug(:dispatch)
 
+  forward("/v1", to: ExAuction.Login.V1.Receiver)
```

Comment:
I would prefer not to route everything through Login's Receiver

Reply:
It's just a temp setup, there will be the protected endpoint, too. It's simply commented now :P

Reply:
ok

Reply:
what I mean is that I would prefer /v1/login to route to the Login receiver rather than all /v1

suppose we need to add other routes in the future